src/web/mjs/engine/Storage.mjs

```diff
@@ -30,6 +30,7 @@ export default class Storage {
         this.fs = require( 'fs' );
         this.path = require( 'path' );
         this.config = this.path.join( electron.remote.app.getPath( 'userData' ), 'hakuneko.' );
+        this.root = electron.remote.app.getPath( 'userData');
```

Comment:
couldn't use `this.config` since it contains hakuneko in the path. i don't like rewriting original paths or duplicating variable values, this needs to change